### PR TITLE
Improves docs on interfaces and unions

### PR DESCRIPTION
**Pull request by RobAWilkinson**

### CODE REVIEW

File: source/graphql-tools/resolvers.md [hunk 1]
```diff
@@ -86,10 +86,30 @@ So, in the example query above, the `name` and `title` fields wouldn't need a re
...
 For example, if you have a `Vehicle` interface type with members `Airplane` and `Car`:
 
+You could specify the schema like so
+
+```
+interface Vehicle {
+  max_speed: Int
+}
+
+type Airplane implements Interface {
```
Comment (helfer) [inline source/graphql-tools/resolvers.md]: this should say `Vehicle`

File: source/graphql-tools/resolvers.md [hunk 2]
```diff
@@ -86,10 +86,30 @@ So, in the example query above, the `name` and `title` fields wouldn't need a re
...
+  max_speed: Int
+  engines: Int
+}
+
+type Car implements Interface {
```
Comment (helfer) [inline source/graphql-tools/resolvers.md]: same here

**Comment by RobAWilkinson** [comment]
Those fixes are in!

I have not used `__ofType` @helfer I scanned the graphql-js repo for it. I need to do some research and maybe come up with a real world example using it on our current project before I feel confident writing docs on it.

File: source/graphql-tools/resolvers.md [hunk 3]
```diff
@@ -98,7 +120,7 @@ const resolverMap = {
         return 'Airplane';
       }
 
-      if(data.licensePlate){
+      if(data.license_plate){
```
Comment (stubailo) [inline source/graphql-tools/resolvers.md]: The convention for GraphQL is to use `camelCase`. Let's stick to that in the docs. GitHunt only uses `snake_case` for places where we are reproducing the GitHub API.

**Comment by RobAWilkinson** [comment]
@stubailo camelCased!

**Comment by stubailo** [comment]
Thanks a lot!

**Pull request merged**